scripts/install.sh

```diff
@@ -159,16 +158,22 @@ celery worker -A ${INVENIO_WEB_INSTANCE}.celery -l INFO &
 # sphinxdoc-start-celery-worker-end
 
 # sphinxdoc-populate-with-demo-records-begin
-echo '{"title":"Invenio 3 Rocks", "recid": 1}'| \
-    python manage.py records create
+echo "from invenio_db import db; \
+from invenio_records import Record; \
+Record.create({'title': 'Invenio 3 rocks'}, \
+  id_='4b0714f6-9fe4-43d3-a08f-38c2b309afba'); \
+db.session.commit()" | python manage.py shell
 # sphinxdoc-populate-with-demo-records-end
```

Comment:
Some notes: 

- Using low-level Python API, which works, but which may not be the prettiest for the first-seen end-user installation example. 

- Would be nice to have some high-level CLI API such as `python manage.py records create` that would also take care of minting record IDs in PIDStore... though beware about mutual package dependencies.  

- Could switch to curl'ing JSON file to the upload REST APIs later, perhaps.

- In any case, loading records via CLI command, without having any PIDs back, does not seem to be very reuse friendly for those records... let's think about some consistent CLI API vs REST API options.